packages/@okta/vuepress-site/docs/guides/implement-grant-type/main/authcode/install-sdk.md

```diff
@@ -2,4 +2,4 @@
 
 Okta recommends using existing libraries and OAuth 2.0 helper methods to implement your authentication flow. You can use one of Okta's many language and framework SDKs or an open-source library if an appropriate Okta SDK is not available. See [Languages & SDKs overview](/code/) for a list of Okta SDKs that you can download to start using with your app.
 
-Refer to [Add and configure packages](/docs/guides/sign-into-web-app/-/configure-packages) for instructions on how to install and use Okta back-end framework SDKs. You can download Okta sample apps to see how the SDKs are used in your app's framework. See [Examples](#examples) for a list of sample apps.
+Refer to [Add and configure packages](/docs/guides/sign-into-web-app/aspnet/main/#add-and-configure-packages) for instructions on how to install and use Okta back-end framework SDKs. You can download Okta sample apps to see how the SDKs are used in your app's framework. See [Examples](#examples) for a list of sample apps.
```

Comment:
I think this is ok for now (hardcoding `aspnet` will avoid the left-nav bug). But in the future, if we implement the global language-selector for the site, all this hardcoding will be an issue.

Reply:
OK. I'm going to try to get the /-/ bug fixed asap by the agency I'm hiring, so hopefully that will be OK soonish. We can always do a search and replace to get rid of all the hardcoded URLs when the time comes.